pkg/sql/schemachanger/scexec/exec_mutation.go

```diff
@@ -70,7 +67,7 @@ func executeDescriptorMutationOps(ctx context.Context, deps Dependencies, ops []
 
 type mutationVisitorState struct {
 	c                     Catalog
-	checkedOutDescriptors catalog.DescriptorIDSet
+	checkedOutDescriptors nstree.Map
```

Comment:
I need this to be able to log side effects. Ultimately this is also what prompted the introduction of `descBuilder` and friends.